src/FriendSearchBar.tsx

```diff
@@ -12,7 +12,6 @@ import AutoSuggest, {
 const FriendSearchBar = () => {
   const [value, setValue] = useState("");
   const [suggestions, setSuggestions] = useState(Array<User>());
-  const [loading, setLoading] = useState(false);
```

Comment:
Thanks for removing this. I had originally planned on doing something with the loading state, but after that never materialized, I didn't get around to removing it.